bin/update_i18n.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :branch,  "The target branch",:type => :string,  :required => false
+  opt :dry_run, "",                 :type => :boolean, :default => true
```

Comment:
We now have `ManageIQ::Release.common_options`, which is preferable and will automatically give you repo and repo_set support.